requirements.in

```diff
@@ -12,6 +12,7 @@ ipytest
 pillow
 jinja2
 numba
+scipy
```

Comment:
so we do need scipy now for the CI to run? I think the run without it was cancelled.